js/api_openlayers.js

```diff
@@ -361,20 +396,26 @@ function clearMap() {
   }
 }
 
```

Comment:
I don't think we need to modify this function. If global variable `latest` will be set everything will work as is.